mmf/configs/zoo/models.yaml

```diff
@@ -204,11 +204,11 @@ m4c:
   textvqa:
```

Comment:
Perhaps not directly related to this PR, but the model `visual_bert.finetuned.vqa2.from_coco_train` is broken because the hashcode is wrong.

It should be `e786351503f0e19abd916cc9063b427adc60a0f98f6686e58b296672e90b74c8` instead (now it is `e786351503f0e19abd916cc9063b427adc60a0f98f6686e58b296672e90b74c`; the last '8' was missing).

Reply:
Fixing in other PR on VisualBERT.